docs/api/ShallowWrapper/dive.md

```diff
@@ -2,6 +2,8 @@
 
 Shallow render the one non-DOM child of the current wrapper, and return a wrapper around the result.
 
+NOTE: `dive` would throw an error if the wrapper contains more than one child, or if the (only one) child isn't a custom React component.
```

Comment:
This note is a duplicate of the one that appears right below it.

Reply:
I just made a change by amending the commit, please check.